docs/section22/Rule22-17.md

```diff
@@ -0,0 +1,52 @@
+
+# CHW&CW - Rule 22-17  
+
+**Schema Version:** 0.0.25    
+**Mandatory Rule:** True  
+**Rule ID:** 22-17  
+**Rule Description:** The baseline heat rejection device shall have an efficiency of 38.2 gpm/hp  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** G3.1.3.11
+**Data Lookup:** None  
+**Evaluation Context:** Each HeatRejection 
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b.
+2. B-RMR is modeled with heat rejection loop
+
+**Function Call:**  
+
+1. get_heat_rejection_loops_connected_to_baseline_systems()
+
+**Applicability Checks:**  
+
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
+
+- check applicability for each HeatRejection in the model: `for heat_rejection in B_RMI.heat_rejections:`
+
+  - check if heat_rejection fluid_loop is one of the applicable loops, if so, continue to the rule logic: `if heat_rejection.loop.id in heat_rejection_loop_ids: CONTINUE TO RULE LOGIC`
```

Comment:
Should we check it for each heat rejection object attached to baseline CW loops? (Same comment as on 2-13, etc.)

Reply:
yes, this does check for each.  (same response as for the others)